### PR TITLE
Added onLongClick binding adapter

### PR DESCRIPTION
**Pull request by remvst**
Gives us basic support for long press

### CODE REVIEW

**Comment by remvst** [comment]
@mproberts @michaelcosta poke

File: library/src/main/java/com/github/mproberts/rxdatabinding/bindings/ViewDataBindings.java [hunk 1]
```diff
@@ -98,6 +98,17 @@ public void accept(Boolean value) throws Exception {
         }, view, newValue);
     }
 
+    @BindingAdapter("android:onLongClick")
+    public static void bindAndroidOnLongClick(final View view, final Runnable listener) {
+        view.setOnLongClickListener(new View.OnLongClickListener() {
+            @Override
+            public boolean onLongClick(View v) {
+                listener.run();
+                return true;
```
Comment (michaelcosta) [inline library/src/main/java/com/github/mproberts/rxdatabinding/bindings/ViewDataBindings.java:109]: do we always want to return true here? are there any scenarios that we would not want to handle the long click

**Comment by remvst** [comment]
@michaelcosta updated to take in a `Callable`, that way it's the responsibility of whoever is using the adapter to consume the click or not.

@mproberts poke

**Comment by remvst** [comment]
@mproberts poke

**Comment by remvst** [comment]
@mproberts poke

File: library/src/main/java/com/github/mproberts/rxdatabinding/bindings/ViewDataBindings.java (hunk 1 above)
Comment (mproberts) [reply library/src/main/java/com/github/mproberts/rxdatabinding/bindings/ViewDataBindings.java:109]: If you wanted a complex behaviour I don't think you would manage it at the binding layer. The view model should be dumb in these cases and not define view behaviour

**Pull request merged**